metadata.json

```diff
@@ -0,0 +1,10 @@
+{
+  "name": "nubis-puppet-eip",
+  "version": "0.0.1",
+  "author": "nubis",
```

Comment:
I have seen this before, but IMO we should put the actual author in here, or at lease a mailing list. This way a downstream user would have an easier time of requesting support or getting involved. We can discuss later, this is a non-blocking concern (and not specific to this project).